go-selinux/label/label_selinux_test.go

```diff
@@ -34,14 +34,15 @@ func TestInit(t *testing.T) {
 		t.Log("InitLabels Disabled Failed")
 		t.FailNow()
 	}
+	selinux.SetEnabled()
 	testUser := []string{"user:user_u", "role:user_r", "type:user_t", "level:s0:c1,c15"}
 	plabel, mlabel, err = InitLabels(testUser)
 	if err != nil {
 		t.Log("InitLabels User Failed")
 		t.Fatal(err)
 	}
 	if plabel != "user_u:user_r:user_t:s0:c1,c15" || (mlabel != "user_u:object_r:container_file_t:s0:c1,c15" && mlabel != "user_u:object_r:svirt_sandbox_file_t:s0:c1,c15") {
-		t.Log("InitLabels User Match Failed")
+		t.Log("InitLabels User Match Failed", plabel, mlabel)
```

Comment:
you need `Logf("InitLabels User Match Failed %s, %s", plabel, mlabel)`

Reply:
Fixed